ICSharpCode.Decompiler.Tests/ICSharpCode.Decompiler.Tests.csproj

```diff
@@ -13,7 +13,10 @@
     <RuntimeIdentifier Condition="$(IsWindowsARM64) == true">win-arm64</RuntimeIdentifier>
 
     <IsPackable>false</IsPackable>
-    <StartupObject>AutoGeneratedProgram</StartupObject>
+    
+    <GenerateTestingPlatformEntryPoint>true</GenerateTestingPlatformEntryPoint>
```

Comment:
This is true by default. It's strange if you don't get the generation automatically and there may be a bug somewhere.

Reply:
Yes, I know - this is just to make it explicit that we want the override for the generated one.